src/client/.storybook/webpack.config.js

```diff
@@ -1,22 +1,26 @@
 const path = require('path')
 
 module.exports = (baseConfig, env, defaultConfig) => {
+  defaultConfig = require('../config-overrides')(defaultConfig, env)
+
   defaultConfig.module.rules.push({
     test: /\.(ts|tsx)$/,
-    loader: require.resolve('awesome-typescript-loader')
+    loader: require.resolve('awesome-typescript-loader'),
   })
-  defaultConfig.resolve.extensions.push('.ts', '.tsx')
+  defaultConfig.resolve.extensions.push('.ts', '.tsx', '.js')
   defaultConfig.resolve.alias = {
     'rt-components': path.resolve(__dirname, '../src', 'rt-components'),
     'rt-util': path.resolve(__dirname, '../src', 'rt-util'),
     'rt-types': path.resolve(__dirname, '../src', 'rt-types'),
     'rt-actions': path.resolve(__dirname, '../src', 'rt-actions'),
     'rt-storybook': path.resolve(__dirname, '../src', 'rt-storybook'),
-    'rt-style-guide': path.resolve(__dirname, '../src', 'rt-style-guide'),
+    'rt-styleguide': path.resolve(__dirname, '../src', 'rt-styleguide'),
     'rt-themes': path.resolve(__dirname, '../src', 'rt-themes'),
+    'rt-theme': path.resolve(__dirname, '../src', 'rt-theme'),
     ui: path.resolve(__dirname, '../src', 'ui'),
```

Comment:
I do not think we should reference UI, Shell and system using aliases. I will put a task to remove these in a seperate PR

Reply:
Sounds good to me.